src/test/codegen/repeat-trusted-len.rs

```diff
@@ -8,6 +8,6 @@ use std::iter;
 // CHECK-LABEL: @repeat_take_collect
 #[no_mangle]
 pub fn repeat_take_collect() -> Vec<u8> {
-// CHECK: call void @llvm.memset.p0i8.i{{[0-9]+}}(i8* {{(nonnull )?}}align 1{{.*}} %{{[0-9]+}}, i8 42, i{{[0-9]+}} 100000, i1 false)
+// CHECK: call void @llvm.memset.p0i8.i{{[0-9]+}}(i8* {{(nonnull )?}}align 1{{.*}} %.fca.0.extract.i.i.i.i.i.i.i.i, i8 42, i{{[0-9]+}} 100000, i1 false)
```

Comment:
The codegen output contained that, so I replaced it, but I have no idea if that breaks the test or not. :upside_down_face: